src/ImageSharp/Common/Helpers/SimdUtils.HwIntrinsics.cs

```diff
@@ -66,9 +66,9 @@ public static void Shuffle4Reduce(
             ref Span<float> destination,
             [ConstantExpected] byte control)
         {
-            if ((Vector512.IsHardwareAccelerated && Vector512Utilities.SupportsShuffleFloat) ||
-                (Vector256.IsHardwareAccelerated && Vector256Utilities.SupportsShuffleFloat) ||
-                (Vector128.IsHardwareAccelerated && Vector128Utilities.SupportsShuffleFloat))
+            if ((Vector512.IsHardwareAccelerated && Vector512_.SupportsShuffleNativeFloat) ||
```

Comment:
Some of these could be supported on other platforms if you passed in the indexes rather than a `byte control`.

There's some other functions like that throughout the `SimdUtils.HwIntrinsics.cs` file as well which could be impactful to Arm64, Wasm, and Android; but I've not looked at them in depth and I'm guessing many are larger refactorings to clean up.

----

Given these are marked as `AggressiveInlining` and `byte control` is expected to be a constant, the JIT "might" (just double check the codegen) be able to do the right thing for something like `Vector128.Shuffle(vector, Vector128.Create((control & 3), ((control >> 2) & 3), ((control >> 4) & 3), ((control >> 6) & 3)))` since that should itself break down to 4 constants which the JIT can recognize